### PR TITLE
[6413] Update bulk recommend content

### PR DESCRIPTION
**Pull request by darokel**
### Context

https://trello.com/c/q5zgzL8A/6413-update-bulk-recommend-qts-content

### Changes proposed in this pull request

Updates the content for the bulk recommendation page to be in line with the prototype.

Before:

Screenshot 2023-12-28 at 11 23 31

After:

Screenshot 2023-12-28 at 11 21 47

### Guidance to review

This is part 1. The content has been modified to suit the current implementation. There is a follow up ticket to include the empty template content as we'll need to adjust the recommendation logic to be able to work with a different csv template file.

### CODE REVIEW

**Comment by darokel** [comment]
[Update bulk recommend QTS content](https://trello.com/c/q5zgzL8A/6413-update-bulk-recommend-qts-content)

File: app/views/bulk_update/recommendations_uploads/new.html.erb [hunk 1]
```diff
@@ -9,15 +9,62 @@
...
+
+      <p class="govuk-body">
+        Open the file in <span class="app-nowrap">spreadsheet software</span> (for example, <span class="app-nowrap">Microsoft Excel</span>).
+      </p>
+
+      <p class="govuk-body">In the column ‘Date standards met’ add the date the trainee met QTS or EYTS standards.</p>
+
+      <p class="govuk-body">
+        The date must be written in the format <span class="app-nowrap">‘DD/MM/YYYY’</span>.
+        For example, if the trainee met the standard on <span class="app-nowrap">20 June 2023</span>, write <span class="app-nowrap">‘20/07/2023’</span>.
```
Comment (stevehook) [inline app/views/bulk_update/recommendations_uploads/new.html.erb:53]:
```suggestion
        For example, if the trainee met the standard on <span class="app-nowrap">20 June 2023</span>, write <span class="app-nowrap">‘20/06/2023’</span>.
```

**Review by stevehook — APPROVED** [review]
One small copy correction needed. Otherwise it all looks good to me.

**Review by defong — APPROVED** [review]

**Comment by andrewscrivener** [comment]
Looking good to me too @darokel Thanks :shipit:

**Pull request merged**